patientsearch/src/js/components/MyPatientsCheckbox.js

```diff
@@ -46,12 +46,13 @@ export default function MyPatientsCheckbox({ shouldDisable, shouldCheck, changeE
           }}
         />
       }
-      label="My Patients"
+      label={label || "My Patients"}
```

Comment:
minor nit, but having a fallback in two places (config.py and here) will likely lead to confusion...

Reply:
thank you, fixed.